physical/raft/fsm.go

```diff
@@ -38,10 +38,11 @@ const (
 
 var (
 	// dataBucketName is the value we use for the bucket
-	dataBucketName   = []byte("data")
-	configBucketName = []byte("config")
-	latestIndexKey   = []byte("latest_indexes")
-	latestConfigKey  = []byte("latest_config")
+	dataBucketName     = []byte("data")
+	configBucketName   = []byte("config")
+	latestIndexKey     = []byte("latest_indexes")
+	latestConfigKey    = []byte("latest_config")
+	desiredSuffrageKey = []byte("desired_suffrage")
```

Comment:
I know I suggested using a dedicated key to store desired suffrage, but maybe we should instead have a `local_node_config` key and store a JSON blob?  Gives us a little more flexibility going forward.

Reply:
Oh, looks like it's already a PB.  Then all the more reason to use a more general key name.

Reply:
Great suggestion. Renamed the key name to `local_node_config`. The entry is a proto message. It should accommodate future values easily.